advisor/src/main/kotlin/advisors/Osv.kt

```diff
@@ -163,8 +163,6 @@ private fun createRequest(pkg: Package): VulnerabilitiesForPackageRequest? {
         )
     }
 
-    // TODO: Handle C++ projects and / or requesting vulnerabilities by commit-ish.
```

Comment:
commit-message:

> here is no ecosystem documented at [1] that would support "requesting
vulnerabilities by commit-ish"

commit-ish is ecosystem agnostic

> , nor is it supported to generally handle
C++ projects independently of a package / dependency manager.

The following does not count?

```
curl -X POST -d '{"commit": "6879efc2c1596d11a6a6ad296f80063b558d5e0f"}' "https://api.osv.dev/v1/query"
```

> On top of
that, support for Conan (which is a package manager for C / C++) has been
added 

This reads as if all C++ projects would use Conan...

Reply:
As you didn't comment on #6536 I simply have it a shot 😉 Thanks for the clarification, I've instead reworded the `TODO` statement now.